moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -54,6 +56,26 @@ const ProjectComponentsList = ({
     setIsClickedComponentRelated(false);
   };
 
+  /* Component link copy button */
+  const [copiedUrl, setCopiedUrl] = useState(null);
+
+  const copyLinkToClipboard = () => {
+    const currentUrl = window.location.href;
```

Comment:
just thinking out loud—i'm wondering if it would be better to generate the link string here rather than trusting that the current browser URL is always going to be what we want, but i can't see of a reason why that would be an improvement. this looks good!

Reply:
This is a good point because there is a difference between capturing the current browser URL (which may hold more search params in the future) and only the relevant info needed to focus on the component. 🤔

Reply:
Yes, but hard to say if in the future we would want future params or not. Definitely I prefer this as-is 😎